### DISCUSSION TITLE
vs code 擴充套件 Vibrancy Continued

### DISCUSSION THREAD

**Discussion by danwan1027** [1101-GS4719-C/Python-Community-GS4719-C#716] [General]
下載這個套件後，可以讓vs code 的背景變成半透明
[
![](https://github.com/illixion/vscode-vibrancy-continued/raw/HEAD/screenshot.png)
](url)
讓編寫程式時的畫面看起來賞心悅目
附上官方文件說明：https://marketplace.visualstudio.com/items?itemName=illixion.vscode-vibrancy-continued

**Comment by danwan1027** [comment]
用了他之後我越來越常打開我的 vs code

**Comment by a45320614** [comment]
寫程式一段時間後常常會有視覺疲乏的情況
這個套件可以協助提升視覺體驗，非常有幫助呢!

**Comment by Yunhans** [comment]
每天打開vscode都開心

**Comment by AnAnistired** [comment]
這樣變得好美喔，不知道還會不會出更多自訂的主題顏色的套件

**Comment by williamyen0623** [comment]
哇！這樣開其他參考資料的時候，能更清楚知道視窗在哪裡，真香

**Comment by manipulateed** [comment]
這樣提升了編程環境的美化，高級感都出來了。

**Comment by Pietra1226** [comment]
這樣子打code多一點樂趣了(?)

**Comment by evanchou1204** [comment]
下載完視窗看的好舒服，雖然這樣不一定會讓我想常常打開vscode。QQ

**Comment by ruthchen1020** [comment]
真的得很舒服，可以隱約的看到漂亮的桌面~

**Comment by StevenWanglolz** [comment]
這是vscode的新造型嗎? 好票釀
